R/readepi_helpers.R

```diff
@@ -130,3 +128,40 @@ fingertips_get_args <- function(args_list =
     parent_area_type_id = parent_area_type_id
   )
 }
+
+#' Check if the base_url argument is specified as a correct URL
+#'
+#' @param base_url the URL to the HIS of interest
+#'
+#' @keywords internal
+#'
+url_check <- function(base_url) {
```

Comment:
Please add some tests for this.

Reply:
I noticed this when I was building the server class in the new implementation.

The `grepl("^https?://", base_url)` option works fine. Will update the function based on this.